ViewModel/Kiyoh.php

```diff
@@ -74,15 +74,15 @@ public function getRating(){
     }
 
     /**
-     * @return bool|Variable|string
+     * @return bool|string
      * @throws NoSuchEntityException
      */
     public function getRatingPercentage(){
-        if(!$this->getVariableValueByCode('kiyoh_percentageRecommendation')){
+        if(!$this->getVariableValueByCode('kiyoh_recommendation')){
```

Comment:
If you change the return type of `getVariableValueByCode()` to `?string`, you can change this code to:

```php
public function getRatingPercentage(): ?string 
{
    return $this->getVariableValueByCode('kiyoh_recommendation');
}
```

That would result in the other method to be like this:

```php
public function getVariableValueByCode($code): string
{
    $customVariable = $this->variable->create()
        ->setStoreId($this->storeManager->getStore()->getId())
        ->loadByCode($code);

    return $customVariable instanceof \Magento\Variable\Model\Variable // not sure if this type is correct
        ? $customVariable->getValue()
        : null;
}
```